quick-start-with-tidb.md

```diff
@@ -0,0 +1,303 @@
+---
```

Comment:
These 2 files need to be deleted. And you can check globally whether there are other `<br>` to be updated.